regions/io/core.py

```diff
@@ -71,12 +71,12 @@ class RegionConversionError(ValueError):
 coordsys_mapping = {'DS9': {x: x for x in valid_coordsys['DS9']},
                     'CRTF': {x: x for x in valid_coordsys['CRTF']}
                     }
-coordsys_mapping['CRTF']['geocentrictrueecliptic'] = 'ecliptic'
-coordsys_mapping['CRTF']['fk5'] = 'j2000'
-coordsys_mapping['CRTF']['fk4'] = 'b1950'
-coordsys_mapping['CRTF']['supergalactic'] = 'supergal'
+coordsys_mapping['CRTF']['geocentrictrueecliptic'] = 'ECLIPTIC'
+coordsys_mapping['CRTF']['fk5'] = 'J2000'
+coordsys_mapping['CRTF']['fk4'] = 'B1950'
+coordsys_mapping['CRTF']['supergalactic'] = 'SUPERGAL'
 
-coordsys_mapping['DS9']['geocentrictrueecliptic'] = 'ecliptic'
+coordsys_mapping['DS9']['geocentrictrueecliptic'] = 'ECLIPTIC'
```

Comment:
Lower-case names are used in the [specification document](https://casaguides.nrao.edu/index.php/CASA_Region_Format), so we should use lower case.

Reply:
To quote the specification document:

> 
> Coordinate reference frame:
> 
>     Possible values: J2000, JMEAN, JTRUE, APP, B1950, B1950_VLA, BMEAN, BTRUE, GALACTIC, HADEC, AZEL, AZELSW, AZELNE, AZELGEO, AZELSWGEO, AZELNEGEO, JNAT, ECLIPTIC, MECLIPTIC, TECLIPTIC, SUPERGAL, ITRF, TOPO, ICRS
> 
>     Default: image value

Using upper-case names enabled the use of global coord in the file, so I believe these are intended as upper-case.

Reply:
Ah, that's very valuable information!  Could you add that in a comment above that, referencing the part of the docs that shows that?

Reply:
Now that you point this out, it seems like a huge oversight in our original implementation.  Thanks again for finding it!

Reply:
also, I was wrong in my original statement: the specs _always_ use uppercase for the coordinate reference frames, but lowercase for region types etc.